appendices/editor-in-chief-checks.md

```diff
@@ -0,0 +1,36 @@
+```markdown
+## Editor in Chief checks 
```

Comment:
from slack:
we should ask at least the minimum Python that is not EOL: https://endoflife.date/python

we require that the package be published on pypi preferred or conda or both.
But if it's on conda - on a public channel like bioconda or conda-forge





Filipe Pires Alvarenga Fernandes
  [13 minutes ago](https://pyopensci.slack.com/archives/C046UJT3JBV/p1671121704195099?thread_ts=1671037979.205939&cid=C046UJT3JBV)
IMO we should ask at least the minimum Python that is not EOL: https://endoflife.date/python

Reply:
We might want to consider either adding that packages that depend on `numpy` meet NEP-29 and so have a minimum Python requires that's in sync with their cycle
https://numpy.org/neps/nep-0029-deprecation_policy.html
or maybe just adopt their policy more widely as @ocefpaf suggested in Slack

Reply:
Hi, stepping in to link here the draft of the [scientific python SPEC 00](https://scientific-python.org/specs/spec-0000/) built on top of the nep29. It is still under discussion but it is a good starting point for maintainers.

Reply:
thank you all for this feedback! it's super helpful. I'm chatting with Stefan about the spec as wherever possible, we want to use the specs that they are developing. and this also helps maintainers think about what versions of core tools they support too. @cmarmo thank you so much for directing us to that spec. it's really helpful to have such cross pollination here in our reviews. you rock!

Reply:
ok we don't want to use NEP-29. it's not a community rec. It sounds like the SPEC 00 is what we want to use but i  need a bit of confirmation on timeline for it to be approved, etc. i'll add some language in the docs for now that hopefully works !!

Reply:
i'm also not sure this belongs here. it's pretty specific. maybe it gets adding to packaging guide and it's part of reviewer checks. it just feels too specific for editor checks.